app/models/katello/product_content.rb

```diff
@@ -0,0 +1,28 @@
+module Katello
+  class ProductContent < Katello::Model
+    belongs_to :product, :class_name => 'Katello::Product', :foreign_key => 'product_id'
+    belongs_to :content, :class_name => 'Katello::ProductContent::Content', :foreign_key => 'content_id'
```

Comment:
I don't see this 'Katello::ProductContent::Content'  class defined in the PR, did you forget to add it?

I'm not sure I like the name of that class TBH, any reason to not use "Katello::Content" ?  As i understand it, in candlepin there is 'Content' and 'ProductContent',   with 'ProductContent' just being the 'join' between products and content.

Reply:
Oh, i see it down below in this file.  We typically put these classes in their own files, and i believe rails auto-reloading expects that?

Reply:
Overall, i think this is shaping up, this piece is really my biggest concern (the name of the class, and its location).

Reply:
ProductContent exists b/c it holds the 'enabled' flag for the content - if we don't need to cache that in our db (I guess maybe it doesnt, for our current purpose), then ProductContent can simply be a join table. What do you think?

Reply:
Gotcha. Will make it so.